lighthouse-core/report/html/renderer/details-renderer.js

```diff
@@ -304,16 +304,45 @@ class DetailsRenderer {
     }
 
     return tableLike.headings.map(heading => {
+      let multi;
+      if (heading.multi) {
+        multi = {
+          key: heading.multi.key,
```

Comment:
nit: would be nice to reuse this mapping bit rather than manually map since that's what the original point of the function was, but not big deal

Reply:
done, but had to disable some ts.